linera-views/src/map_view.rs

```diff
@@ -207,6 +207,37 @@ where
     pub fn extra(&self) -> &C::Extra {
         self.context.extra()
     }
+
+    /// Tests if a key is contained in the container.
```

Comment:
Or maybe like in the standard library: "Returns `true` if the map contains a value for the specified key."